addon/templates/components/my-materials.hbs

```diff
@@ -0,0 +1,132 @@
+<div class="material-list">
+  {{#if (gt (get materials "length") 0)}}
+    <span class="course-filter">
+      <select onchange={{action @setCourseIdFilter value="target.value"}}>
+        <option value="">{{t "general.allCourses"}}</option>
+        {{#each this.courses as |courseObj|}}
+          <option
+            selected={{is-equal courseObj.id @courseIdFilter}}
+            value={{courseObj.id}}>
+            {{courseObj.title}}
+          </option>
+        {{/each}}
+      </select>
+    </span>
+
+    <span class="filter">
+      <input
+        placeholder={{t "general.filterPlaceholder"}}
+        type="text"
+        value={{@filter}}
+        oninput={{perform this.setQuery value="target.value"}}>
+    </span>
+
+    <table>
+      <thead>
+        <tr>
+          {{#sortable-th
+            colspan=3
+            sortedAscending=this.sortedAscending
+            sortedBy=(or (eq @sortBy "sessionTitle") (eq @sortBy "sessionTitle:desc"))
+            click=(action "sortBy" "sessionTitle")}}
+            {{t "general.session"}}
+          {{/sortable-th}}
+          {{#sortable-th
+            colspan=3
+            sortedAscending=this.sortedAscending
+            sortedBy=(or (eq @sortBy "courseTitle") (eq @sortBy "courseTitle:desc"))
+            click=(action "sortBy" "courseTitle")}}
+            {{t "general.course"}}
+          {{/sortable-th}}
+          {{#sortable-th
+            colspan=3
+            sortedAscending=this.sortedAscending
+            sortedBy=(or (eq @sortBy "title") (eq @sortBy "title:desc"))
+            click=(action "sortBy" "title")}}
+            {{t "general.title"}}
+          {{/sortable-th}}
+          <th class="hide-from-small-screen">{{t "general.instructor"}}</th>
```

Comment:
The translations for `instructor` doesn't seem to be in common yet.

Reply:
I found it.

https://github.com/ilios/common/blob/master/translations/en-us.yaml#L124

Reply:
That's plural instructor(s). You'll need https://github.com/ilios/frontend/blob/master/translations/en-us.yaml#L141

Reply:
Doh!